posts/2017-06-01-may.md

```diff
@@ -0,0 +1,75 @@
+---
+menu:
+  main:
+    parent: "Posts"
+title: Dev Diary (Mayh 2017)
+author: Corey Montella
+date: 2017-06-1
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://witheve.com))_
+
+In May we released Preview 3 of v0.3, published some new documentation, and had a meetup.
+
+## Preview 3 of v0.3
+
+### The Eve Syntax Returns
+
+The parser for the Eve syntax returns in Preview 3. While the parser is included again, there’s still no editor bundled, so parsing a program written in the Eve syntax can accomplished in JavaScript like so:
+
+```javascript
+import {Program} from "witheve";
+
+let program = new Program("program name");
+
+program.load(`
+# An eve program
+
+This is an Eve program
+
+~~~
+search
+  [#person name: "Jeff"]
+
+bind
+  [#ui/text text: "Hi Jeff"]
+~~~
+`)
+``` 
+
+Where `some eve code` is a complete multiline Eve document (i.e. the contents of a *.eve file). A second method is to download [eve-starter](https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fwitheve%2Feve-starter&sa=D&sntz=1&usg=AFQjCNHEFK6uF3Udjrz51ipbaJv4j6g1BA), and put your *.eve file in the “programs” directory. Then launch the program switcher (run npm start in the eve-start directory, then direct your browser to localhost:8000), and you will see your program listed in the left panel.
+
+We also updated the [example Eve applications](https://github.com/witheve/eve-starter/tree/master/programs) for v0.3. This entailed removing any references to databases, and updating records to use the new namespaced tags. We haven’t documented the various watchers yet (e.g. html, svg, events) so if you’re curious how to use these, you can use the example programs as a reference for now.
```

Comment:
we are currently working on the documentation for watchers, so i think this should say something like "We're in the process of documenting the various watchers as we speak (eg ...), but if you're curious how to use them and don't want to wait, ..."